src/App.test.js

```diff
@@ -1,9 +1,45 @@
 import React from "react";
 import { render, screen } from "@testing-library/react";
 import App from "./App";
+import userEvent from "@testing-library/user-event";
 
-test("renders the homepage", () => {
+describe("<App />", () => {
+it("renders the homepage", () => {
```

Comment:
```suggestion
describe("<App />", () => {
  it("renders the homepage", () => {
```

Esse arquivo todo está identado errado: os its pertencentes aos discribes, ficam dentro deles, revise o arquivo todo!